README.md

```diff
@@ -31,21 +43,44 @@ methods made available via:
 import org.scalacheck.ops._
 ```
 
-## scalacheck-ops_1-13
+# Installation
+
+### Compatibility
 
-The additional `scalacheck-ops_1-13` library is compiled against 
-ScalaCheck 1.13.x branch because it contains some binary 
-incompatibilities. Specifically, when mixing this library with ScalaTest
-3.x you might notice the following exception:
+**NOTE** Version 2.x and above **requires** JDK >8 and Scala >2.11 as this 
+library expects the java.time standard library module.
+
+**NOTE** As of `scalacheck-ops` 2.x and above, the version of scalacheck is always
+included in the artifact name. Prior to this change, scalacheck-ops with
+no version suffix would pull in ScalaCheck version 1.12.6. 
+
+| Version Limit | ScalaCheck | Artifact Name       | Supported JDK | Supported Scala |
+| :-----------: | :--------: | :-----------------: | :-----------: | :-------------: |
+| 1.x < x < 2.0 |     1.12.6 | scalacheck-ops      | 6 & 8         | 2.10 & 2.11     |
```

Comment:
Is there a case for `x < 1.x` ?

Reply:
I'm pretty certain nobody uses it. The only version would be 0.1.1, which is probably the same as this range, so I'll remove the lower bound.